src/admin-views/settings/tickets-commerce/gateways.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * TEmplate to display a list of gateway options.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
+ * @var array                        $gateways          Array of gateway objects.
+ */
+
+$manager = tribe( TEC\Tickets\Commerce\Gateways\Manager::class );
+
+$brand_info = [
+    'paypal' => [
+        'logo_url' =>  Tribe__Tickets__Main::instance()->plugin_url . 'src/resources/images/admin/paypal_logo.png',
+        'subtitle' => 'Enable payments through PayPal, Venmo, and credit card',
+    ],
+    'stripe' => [
+        'logo_url' =>  Tribe__Tickets__Main::instance()->plugin_url . 'src/resources/images/admin/stripe-logo.png',
+        'subtitle' => __( 'Enable credit card payment and afterpay' ),
+    ],
+];
+
+$gateways = $manager->get_gateways();
+?>
+<div class="tec-tickets__admin-settings-tickets-commerce-gateways">
```

Comment:
Make the main piece one template and the items a separate one.

Reply:
Will do.